src/index.ts

```diff
@@ -135,6 +135,14 @@ export type DecodedRawTransaction = {
   vout: TxOut[]
 }
 
+export type GetDescriptorResult = {
```

Comment:
nit: could you move that under `GetAddressInfoResult` to keep the alphabetical order?